commands/check-status.sh

```
@@ -0,0 +1,67 @@
+echo ""
+
+# Check current directory
+echo "Current directory: $(pwd)"
+echo ""
+
+# Check current git branch and status
+if [ -d ".git" ]; then
+    echo "=== Git Status ==="
+    echo "Current branch: $(git branch --show-current)"
+    echo "Last commit: $(git log -1 --oneline)"
+    echo "Remote tracking: $(git branch -vv | grep \* | awk '{print $4}' | sed 's/\[//g' | sed 's/\]//g')"
```

Shellcheck (0.10.0)

[warning] Grep uses regex, but this looks like a glob.

(SC2063)